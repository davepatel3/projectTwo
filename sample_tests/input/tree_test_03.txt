float
allocate
98.743
0
allocate
4.63954
1
set_parent
2
0
peek_nodes
allocate
-87.3388
2
remove
2
allocate
10.2706
0
remove
3
allocate
-33.0261
1
allocate
52.6138
1
ref_node
2
set_parent
3
0
allocate
-49.5401
1
remove
2
remove
3
remove
1
allocate
-94.5876
0
remove
2
allocate
-4.68341
0
allocate
-19.3044
0
allocate
-15.6217
3
allocate
-81.3092
3
ref_node
1
set_parent
1
4
peek_nodes
allocate
-65.5238
0
peek_nodes
ref_node
1
ref_node
3
allocate
-16.9027
3
ref_node
4
remove
6
allocate
-4.9234
2
remove
4
allocate
-21.2368
2
allocate
-97.2268
6
allocate
82.4587
1
peek_nodes
remove
7
allocate
-80.33
3
ref_node
7
remove
1
allocate
97.6265
5
peek_nodes
allocate
-88.8055
4
allocate
-41.694
3
allocate
-91.8407
0
allocate
25.4309
7
allocate
44.464
11
set_parent
5
8
allocate
-68.9855
11
allocate
55.0934
13
remove
5
set_parent
11
3
peek_nodes
ref_node
4
allocate
-41.0447
12
allocate
46.4372
8
allocate
25.3013
8
remove
10
remove
13
allocate
65.5872
1
peek_nodes
allocate
63.4003
5
allocate
-18.0643
5
allocate
-20.0037
3
remove
8
allocate
-92.272
4
allocate
9.49807
9
set_parent
16
0
ref_node
12
allocate
-72.7783
3
allocate
94.1197
2
allocate
82.3397
10
set_parent
7
11
set_parent
13
0
allocate
66.1235
11
allocate
-41.9266
3
quit
