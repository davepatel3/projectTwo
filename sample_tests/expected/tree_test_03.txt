allocate
1
allocate
2
set_parent
peek_nodes
0: 0
1: 98.743
2: 4.63954
allocate
3
remove
allocate
3
remove
allocate
2
allocate
3
ref_node
-33.0261
parent: 1
children: 
set_parent
allocate
4
remove
remove
remove
allocate
2
remove
allocate
3
allocate
4
allocate
1
allocate
2
ref_node
-15.6217
parent: 3
children: 
set_parent
peek_nodes
0: 0
1: -15.6217
2: -81.3092
3: -4.68341
4: -19.3044
allocate
5
peek_nodes
0: 0
1: -15.6217
2: -81.3092
3: -4.68341
4: -19.3044
5: -65.5238
ref_node
-15.6217
parent: 4
children: 
ref_node
-4.68341
parent: 0
children: 2 
allocate
6
ref_node
-19.3044
parent: 0
children: 1 
remove
allocate
6
remove
allocate
1
allocate
4
allocate
7
peek_nodes
0: 0
1: -21.2368
2: -81.3092
3: -4.68341
4: -97.2268
5: -65.5238
6: -4.9234
7: 82.4587
remove
allocate
7
ref_node
-80.33
parent: 3
children: 
remove
allocate
1
peek_nodes
0: 0
1: 97.6265
2: -81.3092
3: -4.68341
4: -97.2268
5: -65.5238
6: -4.9234
7: -80.33
allocate
8
allocate
9
allocate
10
allocate
11
allocate
12
set_parent
allocate
13
allocate
14
remove
set_parent
peek_nodes
0: 0
1: [recycled]
2: -81.3092
3: -4.68341
4: -97.2268
5: [recycled]
6: -4.9234
7: -80.33
8: -88.8055
9: -41.694
10: -91.8407
11: 25.4309
12: 44.464
13: -68.9855
14: 55.0934
ref_node
-97.2268
parent: 6
children: 8 
allocate
1
allocate
5
allocate
15
remove
remove
allocate
10
peek_nodes
0: 0
1: -41.0447
2: -81.3092
3: -4.68341
4: -97.2268
5: 46.4372
6: -4.9234
7: -80.33
8: -88.8055
9: -41.694
10: 65.5872
11: 25.4309
12: 44.464
13: [recycled]
14: [recycled]
15: 25.3013
allocate
14
allocate
13
allocate
16
remove
allocate
14
allocate
13
set_parent
ref_node
44.464
parent: 11
children: 1 
allocate
5
allocate
15
allocate
8
set_parent
set_parent
allocate
17
allocate
18
quit
